tracing-subscriber/src/reload.rs

```diff
@@ -228,15 +228,12 @@ impl Error {
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        error::Error::description(self).fmt(f)
-    }
-}
-
-impl error::Error for Error {
-    fn description(&self) -> &str {
-        match self.kind {
+        let msg = match self.kind {
             ErrorKind::SubscriberGone => "subscriber no longer exists",
             ErrorKind::Poisoned => "lock poisoned",
-        }
+        };
+        write!(f, "{}", msg)
```

Comment:
slight pref for
```suggestion
        f.pad(msg)
```
there is _slight_ overhead from evaluating format macros that can be avoided when we aren't actually interpolating